catalog/src/main/java/org/polypheny/db/catalog/CatalogImpl.java

```diff
@@ -2236,6 +2292,14 @@ public long addColumn( String name, long tableId, int position, PolyType type, P
             }
         }
 
+        // TODO DL: remove
```

Comment:
Has this todo become obsolete?

Reply:
Yes, I removed it with the commented code.